packages/webpack/src/index.js

```diff
@@ -1 +1 @@
-export { default } from './builder'
+export { WebpackBuilder } from './builder'
```

Comment:
Should be `as BundleBuilder` ?

Reply:
@clarkdo No different. Explicit names help reducing namespace conflicts (Like importing two different builders). Will change it :)

Reply:
I mean https://github.com/nuxt/nuxt.js/pull/4193/files#diff-4f235fc89b94513b32c2cc8b5896cc5aR92 is using `BundleBuilder`, but export name is `WebpackBuilder` 😄

Reply:
You are right! Good catch :D